enclave_wrangler/config.py

```diff
@@ -45,6 +49,16 @@
         f'{", ".join(missing_env_vars)}\n'
         f'{cause_msg}')
 
+FAVORITE_OBJECTS = [
```

Comment:
Moved these 'objects of interest' from `enclave_wrangler;objects.py` into `enclave_wrangler/config.py` as `
FAVORITE_OBJECTS `.